package.json

```diff
@@ -16,8 +17,13 @@
   "bugs": {
     "url": "https://github.com/ashtuchkin/u2f/issues"
   },
+  "scripts": {
+    "build": "./node_modules/.bin/tsc -p ."
+  },
```

Comment:
could you remove this clause? it's not actually a build step, so might be confusing to others.

Reply:
done